### PR TITLE
Comp Sci - The Queue Data Structure

### PR DESCRIPTION
**Pull request by kapnobatai137**
- user submitted a reported asking that the ADT is
- added footnote on ADT

image

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/comp-sci/data-structures-and-algorithms/intro-data-structures/the-queue-data-structure.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/comp-sci/data-structures-and-algorithms/intro-data-structures/the-queue-data-structure.md?ref=e015640ce07128370ac07eaa6ae9b6086400dd23)

File: comp-sci/data-structures-and-algorithms/intro-data-structures/the-queue-data-structure.md [hunk 1]
```diff
@@ -88,4 +88,4 @@ The working principle behind a queue is
 ## Footnotes
 
 [1:ADT]
-This is just an abreviation for the Abstract Data Structure, hence ADT.
\ No newline at end of file
+Abstract Data Type, or ADT, means the data type cannot be implemented (i.e. represented) by itself and it needs some kind of 'support' in order to be used.
```
Comment (nem035) [inline comp-sci/data-structures-and-algorithms/intro-data-structures/the-queue-data-structure.md]: Not sure if this is clear enough 🤔 

"abstract" in CS usually means that you explain that something will have certain capabilities but not the underlying implementation details.

How about we say this?

```suggestion
Abstract Data Type, or ADT, is a definition of a data type (or a data structure) that only mentions what operations are to be performed but not how these operations will be implemented.
```

Comment (kapnobatai137) [reply comp-sci/data-structures-and-algorithms/intro-data-structures/the-queue-data-structure.md]: yeah, it sounds much better.

we also have a footnote on ADT in `The Linked List Data Structure` insight. should I update that one as well?

Comment (nem035) [reply comp-sci/data-structures-and-algorithms/intro-data-structures/the-queue-data-structure.md]: Yeah, let's do it

Comment (kapnobatai137) [reply comp-sci/data-structures-and-algorithms/intro-data-structures/the-queue-data-structure.md]: okay, I'll put both updates in this pr

**Pull request merged**